docs/intro/snapshots.md

```diff
@@ -40,6 +40,43 @@ $ aws s3 sync --delete --no-sign-request  "s3://near-protocol-public/backups/${c
 
 For a faster snapshot download speed, use s5cmd, the download accelerator for S3 written in Go. For download instruction, please see https://github.com/peak/s5cmd.
 
+Next alternative you can use `rclone`. 
+This tool is present in many Linux distributions. There is also a version for Windows.
+And its main merit is multithread.
+You can [read about it on](https://rclone.org)
+
+First, install rclone:
+```
+$ sudo apt install rclone
+```
+Next, prepare config. Because interactivity config very long and you must know meny parameters.
+```
+mkdir -p ~/.config/rclone
+touch ~/.config/rclone/rclone.conf
+```
+... and paste next rows into file rclone.conf 
+```
+[near_s3]
+type = s3
+provider = AWS
+location_constraint = EU
+acl = public-read
+server_side_encryption = AES256
+region = ca-central-1
+```
+Next step very similar with aws-cli. 
+>With two exceptions.
+1. By default rclone use 4 threads. You can change this by adding key `--transfers=X` where X is count of threads.
+2. You can view or not state of downloading. This key is `--progress`
```

Comment:
I feel this is not relevant, and whoever would need to dig into it will learn more about `rclone` separately, so to keep the page short, I will remove this part.

```suggestion
```